rotating_proxies/middlewares.py

```diff
@@ -156,12 +159,17 @@ def _handle_result(self, request, spider):
         proxy = self.proxies.get_proxy(request.meta.get('proxy', None))
         if not (proxy and request.meta.get('_rotating_proxy')):
             return
+        self.stats.set_value('proxies/unchecked', len(self.proxies.unchecked) - len(self.proxies.reanimated))
```

Comment:
This is inconsistent with https://github.com/TeamHG-Memex/scrapy-rotating-proxies/blob/2b3506c22480ab94cd4973bc867b16e6b4bcfb2a/rotating_proxies/expire.py#L15 - there reanimated proxies are considered unchecked; it makes sense to either change both, or keep the same semantics in stats. I'd suggest to keep it for now, and handle change separately, in a different PR, if you have a strong feeling about it.

Reply:
Right, that's where I pulled it from; Which one should we keep then? unchecked - reanimated or just unchecked?  I mean since they are both stored to stats whoever uses it can do the substracting operation themselves.

Reply:
Sorry for a confusing comment! I meant that the way you originally implemented it is the least confusing - stats would match logs :) Would you mind revert it to your original implementation?